ext/net/ops.rs

```diff
@@ -665,6 +666,26 @@ where
   Ok(results)
 }
 
+pub async fn op_set_nodelay<NP>(
+  state: Rc<RefCell<OpState>>,
+  args: NoDelayArgs,
+  _: (),
+) -> Result<(), AnyError> {
+  let rid = args.rid;
+  let nodelay = args.nodelay;
+  let resource: Rc<TcpStreamResource> = state
+    .borrow_mut()
+    .resource_table
+    .get::<TcpStreamResource>(rid)?;
+  resource.set_nodelay(nodelay).await
+}
```

Comment:
Simpler function sig and no destructuring with flattened args:

```suggestion
pub async fn op_set_nodelay<NP>(
  state: Rc<RefCell<OpState>>,
  rid: ResourceId,
  nodelay: bool,
) -> Result<(), AnyError> {
  let resource: Rc<TcpStreamResource> = state
    .borrow_mut()
    .resource_table
    .get::<TcpStreamResource>(rid)?;
  resource.set_nodelay(nodelay).await
}
```